README.md

```diff
@@ -234,6 +234,39 @@ config.  If Fedora requires auth, you can also include that in the URL, e.g.:
      fedora_version: 5
    )
    ```
+#### Pairtree paths in Fedora 4/6
+Fedora 4 and 6.5+ support automatic creation of "pairtree" paths, which means that identifiers are stored in
+a hashed directory structure to avoid excessive child nodes directly in the root node.
+
+Unlike Fedora 4 where this is the default behavior, it has to be enabled via options in Fedora 6.5. This is 
+done by configuring both the count and length of the pairtree algorithm, so it needs to be configured
+in the Fedora adapters so that both sides match.
+
+In the following example, Fedora 6.5 is being started with options to create pairtree paths consisting of 
+4 segments, 2 characters in length:  
+
+```angular2html
+CATALINA_OPTS=-Dfcrepo.home=/fcrepo-home ...
+...
+-Dfcrepo.pid.minter.length=2 -Dfcrepo.pid.minter.count=4
+```
+For the Fedora metadata/storage adapters to correctly translate identifiers into URI's, they should be initialized
+like:
+
+```angular2html
+Valkyrie::Persistence::Fedora::MetadataAdapter.new(
+    connection: ::Ldp::Client.new("http://localhost:8080/fcrepo/rest"),
+    fedora_version: 6,
```

Comment:
Does it make sense for the fedora_version to include minor version in order to distinguish between different versions of Fedora 6 (e.g. `fedora_version >= 6.5`)?

Reply:
@cjcolvar Just FYI, I've been working on some revisions per these suggestions.  I refactored both adapters to consider minor versions, hoping to get that much pushed later today.

Reply:
@cjcolvar  The configuration of pairtree paths is now only valid if Fedora is set to 6.5 or higher.  That changed a lot of comparison operators in both the adapters and specs (since only major numbers were being compared) but I think it's all still correct.

Reply:
Sweet! Did the documentation in the README here get duplicated when it was getting updated?  It looks like it has the same sections twice but with the minor version included in `fedora_version` in the second version.

Reply:
@cjcolvar I was just trying to artificially inflate my contribution stats.  Seriously, I have no idea how I did that or how it escaped my notice.  Probably in the way I squash and edit commits.  Anyway, I just pushed a new version.